src/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Understanding Sessions
+sidebar_order: 4300
+notSupported:
+  - javascript.cordova
+  - javascript.wasm
+description: 'Learn about customizing sessions with the Session Replay SDK.'
+---
+
+In most cases, setting the Replay sample rates will be all you need to do in order to capture recording data.
```

Comment:
```suggestion
In most cases, setting the Replay sample rates will be all you need to do in order to capture recording data you need.
```

Reply:
hmm, that's `you need` twice 🤔

Reply:
what about: 
```suggestion
In most cases, setting the Replay sample rates will be all you need to do in order to capture recording data you care about.
```

?

Reply:
Sounds good!

Reply:
We can shorten it even more: 
In most cases, setting the Replay sample rates will be all you need to do to begin capturing recording data you care about.

Reply:
```suggestion
In most cases, setting the Replay sample rates will be all you need to do to begin capturing recording data you care about.
```